consensus/chronology/chronology.go

```diff
@@ -125,7 +130,9 @@ func (chr *chronology) startRound() {
 		chr.subroundId = srBeforeStartRound
 		return
 	}
-
+	if chr.ash != nil {
+		chr.ash.SetInt64Value(core.MetricCurrentRound, int64(chr.rounder.Index()))
```

Comment:
Because there is not a general way of getting the values for metrics. Right now I found 3 ways of getting data:
- inject AppStatusHandler via constructor / setter method
- poll for changes at a given time (num of connected peers for example)
- register AppStatusHandler as listener (for example bootstrapper.AddSyncStateListener)

Reply:
With the current implementation you could actually create a new handler function, inject it in the polling component, foreach metric update. Or where possible multiple updates in one polling handler. Example:
Instead of injecting the app status handler in the chronology component, you can poll it in the function you created in main and get the current round from the blockchain object.